src/WikiService.Data.Provider.MsSql.Ef/WikiService.Data.Provider.MsSql.Ef.csproj

```diff
@@ -6,4 +6,8 @@
 		<ProjectReference Include="..\WikiService.Data.Provider\WikiService.Data.Provider.csproj" />
 	</ItemGroup>
 
+	<ItemGroup>
+	  <PackageReference Update="LT.DigitalOffice.Kernel" Version="1.1.61" />
```

Comment:
in targets

Reply:
When I use version 1.1.63, I get error in stranger code (in EditArticleCommand).